src/utils/dig.js

```diff
@@ -5,6 +5,8 @@ import providers from './providers.js';
 import { presentTable } from './table.js';
 import { createEmbed } from './embed.js';
 
+const DNSSEC_DISABLED_WARNING_MESSAGE = '\n\n:warning: cd bit set for request, DNSSEC validation disabled';
```

Comment:
```suggestion
const DNSSEC_DISABLED_WARNING_MESSAGE = ':warning: cd bit set for request, DNSSEC validation disabled';
```
I take back what I said about adding the newlines here, sorry (see  below)

Reply:
It's fine haha. I had it that way before in case different formatting is needed alter on